spec/expectations/response_csv/repeat_groups.csv

```diff
@@ -0,0 +1,11 @@
+Form,Submitter,DateSubmitted,ResponseID,Question1,Question2,Question3,Question4,Question5,Question6,RepeatLevel

```

Comment:
I updated these to have hardcoded ids. Previously this file was relying on FactoryGirl's sequence, which is messy with more than one test, and declared an anti-pattern on a couple blogs.